client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -1,9 +1,11 @@
-import { recordTracksEvent } from '@automattic/calypso-analytics';
-import { localizeUrl, useLocale } from '@automattic/i18n-utils';
+//import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { ExternalLinkWithTracking } from '@automattic/components';
 import { useBreakpoint } from '@automattic/viewport-react';
+//import { ExternalLink } from '@wordpress/components';
```

Comment:
I think these comments might be leftover from testing? They should probably be removed

Reply:
Oops!  Yes my mistake. 🤦‍♂️😉